test/integration/actions.test.js

```diff
@@ -112,14 +116,14 @@ test('create, get and delete with parameters an action', t => {
   }).catch(errors)
 })
 
-test('invoke action with fully-qualified name', t => {
+tempTest('invoke action with fully-qualified name', t => {
```

Comment:
Feeds is skipped because alarm isn't installed/included by default during the openwhisk setup process. 

Justin informed me this would be difficult to do and suggested - to not do it.
For action this makes use of whisk.system/utils - which - I found out whisk.system stuff doesn't seem to be present when travis builds. (I was using the script copied from catalog - so this is probably some difference between my personal setup and travis script setup I don't fully understand). 

This should only be skipped when running  the tests with the -i flag. So that was the compromise I did. 

If you want me to not skip them, I need someone to direct me to setup-instructions/part-of-code/etc where whisk.system stuff is being made available. 

And even with that, alarms might still be a problem based on feedback I've gotten from Justin.

Not sure how to proceed. Will wait for your reply.